src/viking.js

```diff
@@ -1,11 +1,99 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength = strength;
+    }
+    attack(){
+        return this.strength
+    }
+    receiveDamage(damage){
+        this.health = this.health - damage
+    }
+};
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name, health, strength){
+        super(health, strength)
+        this.name = name
+    }
+    receiveDamage(damage){
+        this.health = this.health - damage
+        if(this.health > 0){
+            return `${this.name} has received ${damage} points of damage`
+        }
+        else if(this.health <= 0){
+            return `${this.name} has died in act of combat`
+        }
+    }
+    battleCry(){
+        return "Odin Owns You All!"
+    }
+}
+    ;
 
 // Saxon
-class Saxon {}
+class Saxon  extends Soldier {
+    constructor(health, strength){
+        super(health, strength)
+    }
+    receiveDamage(damage){
+        this.health = this.health - damage
+        if(this.health > 0){
+            return `A Saxon has received ${damage} points of damage`
+        }
+        else if(this.health <= 0){
+            return `A Saxon has died in combat`
+        }
+    }}
+
 
 // War
-class War {}
+class War {
+    constructor(){
+        this.vikingArmy = [];
+        this.saxonArmy = [];
+    }
+    addViking(viking){
+        this.vikingArmy.push(viking);
+    }
+    addSaxon(saxon){
+        this.saxonArmy.push(saxon);
+    }
+    vikingAttack(){
+        
+        let aSaxon = this.saxonArmy[Math.floor((this.saxonArmy.length)*Math.random())]
+       
+        
+        let aViking = this.vikingArmy[Math.floor((this.vikingArmy.length)*Math.random())]
+        
+        let damagedSaxon = aSaxon.receiveDamage(aViking.attack())
+
+        if(aSaxon.health <= 0){
+            this.saxonArmy.splice(aSaxon,1)
+        }
+        return damagedSaxon   
+    }
+    saxonAttack(){
+        let aSaxon = this.saxonArmy[Math.floor((this.saxonArmy.length)*Math.random())]
+        let aViking = this.vikingArmy[Math.floor((this.vikingArmy.length)*Math.random())]
+        
+        let damagedViking = aViking.receiveDamage(aSaxon.attack())
+        
+        if(aViking.health <= 0){
+            this.vikingArmy.splice(aViking, 1)
+        }
+        return damagedViking
+    }
+    showStatus(){
+        if(this.saxonArmy.length === 0){
+            return "Vikings have won the war of the century!"
+        }
+        else if(this.vikingArmy.length === 0){
+            return "Saxons have fought for their lives and survived another day..."
+        }
+        else if(this.saxonArmy.length > 0 && this.saxonArmy.length > 0)
```

Comment:
André, here you can use the `else`, since you targeted the two other options with the `if` and the `else if`, you can trigger the `else` operator for the remaining answer.